wildfly-elytron-script-engine/ScriptRoleDecoder.java

```diff
@@ -0,0 +1,27 @@
+import javax.script.Invocable;
+import javax.script.ScriptEngineManager;
+import javax.script.ScriptException;
+import java.util.HashMap;
+import java.util.Set;
+
```

Comment:
There should be a ```package``` statement here. For example, the ```ScriptRoleDecoder``` class could be moved to something like  ```src/main/java/org/wildfly/security/script/engine``` and then the following ```package``` statement could be added here:

```package org.wildfly.security.script.engine```

Reply:
To be consistent with the other modules, the ```wildfly-elytron-script-engine``` directory should be renamed to just ```script-engine```. You'll want to use the full ```wildfly-elytron-script-engine``` name for the ```artifactId``` in the module's pom.xml file.